e2e/fixtures/map-waldflasche-chart-config.json

```diff
@@ -1,6 +1,6 @@
 {
   "state": "CONFIGURING_CHART",
-  "dataSet": "https://environment.ld.admin.ch/foen/nfi/49-19-None-None-44/cube/1",
+  "dataSet": "https://environment.ld.admin.ch/foen/nfi/49-19-44/cube/1",
```

Comment:
👍

Reply:
I think this config can be also enabled [here](https://github.com/visualize-admin/visualization-tool/blob/playwright/app/test/__fixtures/config/int/configs.ts#L31:L36) and changed [here](https://github.com/visualize-admin/visualization-tool/blob/playwright/app/test/__fixtures/config/int/map-waldflasche.json#L5)?

Reply:
Done